framework/resources/Functional/jsonNanInf/negativeCases/queries.json

```diff
@@ -0,0 +1,28 @@
+{
+    "testId": "jsonNaNInf_negative_cases",
```

Comment:
We have categories for negative tests (tests which are expected to fail) - so will need to set that if that's your intension.

Reply:
To clarify, pre-commit runs do not include negative tests. However, we have nightly and release time tests which cover these cases.

Reply:
Thanks for the clarification, the queries.json file has been changed, please verify